src/timedate/timedated.c

```diff
@@ -803,6 +803,7 @@ static int method_set_time(sd_bus_message *m, void *userdata, sd_bus_error *erro
         sd_bus *bus = sd_bus_message_get_bus(m);
         int relative, interactive, r;
         Context *c = userdata;
+        char ctime_buf[32];
```

Comment:
according to the man page this should be 26 bytes long.

That said, I think we shouldn't use ctime/ctime_r at all here, but our own functions: format_timestamp(). It's what we universally use, hence should use here too.

Reply:
manpage says _at least_ 26 - which is not a power of 2 and thus looked just wrong :-)

switched to format_timestamp as asked, PTAL!